tox.ini

```diff
@@ -1,22 +1,17 @@
-# Operator charm: tox.ini
+# Source charm: ./tox.ini
+# This file is managed centrally by release-tools and should not be modified
+# within individual charm repos.  See the 'global' dir contents for available
+# choices of tox.ini for OpenStack Charms:
+#     https://github.com/openstack-charmers/release-tools
 
 [tox]
-envlist = pep8,py3
 skipsdist = True
+envlist = pep8,py3
 # NOTE: Avoid build/test env pollution by not enabling sitepackages.
 sitepackages = False
 # NOTE: Avoid false positives by not skipping missing interpreters.
 skip_missing_interpreters = False
-# NOTES:
-# * We avoid the new dependency resolver by pinning pip < 20.3, see
-#   https://github.com/pypa/pip/issues/9187
-# * Pinning dependencies requires tox >= 3.2.0, see
-#   https://tox.readthedocs.io/en/latest/config.html#conf-requires
-# * It is also necessary to pin virtualenv as a newer virtualenv would still
-#   lead to fetching the latest pip in the func* tox targets, see
-#   https://stackoverflow.com/a/38133283
-requires = pip < 20.3
-           virtualenv < 20.0
+
```

Comment:
I can't comment on the below b/c you didn't edit, but please look at the whitelist_externals - I don't think we need all of those passed through. add-to-archive.py appears to be reference to super early stages of the charmcraft / operator framework stuff ([reference](https://opendev.org/openstack/charm-ceph-iscsi/commit/1c95624727c99631c0be5d703cf97eab2501fb79)) and has carried through to this for quick prototyping but we should clear it up now and not carry some of our things through that are "crufty" already.

Reply:
Done the cleanup